dev-guide/cluster-version-operator/dev/clusteroperator.md

```diff
@@ -18,8 +18,9 @@ When ClusterVersionOperator encounters a ClusterOperator Custom Resource,
 
 - It uses the `.metadata.name` to find the corresponding ClusterOperator instance in the cluster
 - It then waits for the instance in the cluster until
-  - `.status.versions[name=operator].version` in the live instance matches the `.status.version` from the release image and
-  - the live instance `.status.conditions` report available
+  - `.status.versions[name=operator].version` in the live instance matches the `.status.versions.version` from the release image,
+  - the live instance `.status.conditions` report available, and
+  - `.status.versions[name=operator-image].version` in the live instance matches the `.status.versions.version` from the release image
```

Comment:
Instead of mentioning `operator-image` explicitly in this section, can we say that it requires at least the name/version pairs it finds in the release-image manifest to appear in the live instance?  We can still talk about `operator-image` explicitly down in the `What should be the contents of ClusterOperator Custom Resource in /manifests` section, because there we're suggesting a release-image manifest that requests that `versions` entry.